.tx/config

```diff
@@ -1,9 +1,10 @@
 [main]
-host = https://www.transifex.com
-
-[foreman.foreman_ansible]
-file_filter = locale/<lang>/foreman_ansible.edit.po
-source_file = locale/foreman_ansible.pot
-source_lang = en
-type = PO
+host = https://app.transifex.com
 
+[o:foreman:p:foreman:r:foreman_ansible]
+file_filter   = locale/<lang>/foreman_ansible.po
```

Comment:
As @ofedoren pointed out somewhere else and from what I've seen, this should stay `locale/<lang>foreman_ansible.edit.po`